openshift/templates/deploy.dc.yaml

```diff
@@ -128,6 +116,23 @@ objects:
                   value: "5432"
                 - name: POSTGRES_DATABASE
                   value: ${NAME}-${SUFFIX}
+                - name: PROJECT_NAMESPACE
+                  valueFrom:
+                    configMapKeyRef:
+                      name: ${NAME}-global
+                      key: env.project-namespace
+                - name: PATRONI_HEALTH_SUFFIX
```

Comment:
I am not sure `PATRONI_HEALTH_SUFFIX` is the right name for the value. The value is patroni's pod name, right?

Reply:
it's not quite the pod name, since pods are named `patroni-wps-api-dev-0` or `-1` or whatever, and `PATRONI_HEALTH_SUFFIX` just takes the `patroni-wps-api-dev` part. So really it's the patroni cluster name

Reply:
You are right it's a cluster name, my bad. So how about PATRONI_CLUSTER

Reply:
we already have a var called `PATRONI_CLUSTER_NAME` from patroni.yaml template - I'm checking to see if I can just make use of that for my purposes too

Reply:
`PATRONI_CLUSTER_NAME` gets created while running `oc_provision_db.sh` so you would wanna do the same in `oc_deploy.sh`